### PR TITLE
feat: Added `labels.enabled` and `labels.exclude` to config defaults

### LINKED ISSUE #2713: Add application logging labels config to defaults
### Description

In order to support apm custom tags on logs, we need to add `application_logging.forwarding.labels.enabled` and `application_logging.forwarding.labels.exclude` to agent config. Then we need to update the docs. 

Details of implementation:

`application_logging.forwarding.labels.enabled`:
- Description: Toggles whether the agent sends [labels](https://source.datanerd.us/agents/agent-specs/blob/main/Labels.md) as common attributes.
- Type: Boolean
- Default value: `false`
- ENV: `NEW_RELIC_APPLICATION_LOGGING_FORWARDING_LABELS_ENABLED`

`application_logging.forwarding.labels.exclude`:
- Description: A case-insensitive comma and space separated list that contains the labels that SHOULD NOT be added as common attributes.
- Type: Array for config, and string for env var
- ENV: `NEW_RELIC_APPLICATION_LOGGING_FORWARDING_LABELS_EXCLUDE`
- Default value: `null`

[Link to logging spec](https://source.datanerd.us/agents/agent-specs/blob/main/Application-Logging.md#configuration-properties) config properties.

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Description

Updated default configs to accept `application_logging.forwarding.labels.enabled` and `application_logging.forwarding.labels.exclude` to support adding labels to log records. 

Docs website will be updated with these configs in #2716 

## How to Test

Added and updated unit tests

## Related Issues

Closes #2713 
Relates to #2716

### CODE REVIEW

File: test/unit/config/config-env.test.js [hunk 1]
```diff
@@ -653,6 +659,42 @@ test('when overriding configuration values via environment variables', async (t)
     })
   })
 
+  await t.test(
+    'should accept a comman delimited list with spaces for application logging forwarding exclusion list',
```
Comment (jsumners-nr) [inline test/unit/config/config-env.test.js:663]:
```suggestion
    'should accept a comma delimited list with spaces for application logging forwarding exclusion list',
```

File: test/unit/config/config-env.test.js [hunk 2]
```diff
@@ -653,6 +659,42 @@ test('when overriding configuration values via environment variables', async (t)
...
+          'two',
+          'three'
+        ])
+        end()
+      })
+    }
+  )
+
+  await t.test(
+    'should accept a comman delimited list without spaces for application logging forwarding exclusion list',
```
Comment (jsumners-nr) [inline test/unit/config/config-env.test.js:681]:
```suggestion
    'should accept a comma delimited list without spaces for application logging forwarding exclusion list',
```

File: lib/config/default.js [hunk 3]
```diff
@@ -1185,6 +1185,22 @@ defaultConfig.definition = () => ({
       max_samples_stored: {
         formatter: int,
         default: 10000
+      },
+      labels: {
```
Comment (jsumners-nr) [inline lib/config/default.js:1189]: Do we not need an option to supply labels to apply?

Comment (bizob2828) [reply lib/config/default.js:1189]: It's going to read the labels from and already defined configuration `labels`

File: lib/config/default.js [hunk 4]
```diff
@@ -1185,6 +1185,22 @@ defaultConfig.definition = () => ({
...
+        enabled: {
+          formatter: boolean,
+          default: false
+        },
+        /**
+         * A case-insensitive array containing the labels to exclude from log records.
+         */
+        exclude: {
+          formatter: array,
+          default: null
```
[the remark this answers is not in the document]
Comment (svetlanabrennan) [reply lib/config/default.js]: Fixed

**Review by bizob2828 — APPROVED** [review]

**Pull request merged**